### ISSUE TITLE
Removes annotations from the constructor

### ISSUE THREAD

**Issue by PaulWoitaschek** [rharter/auto-value-parcel#93]
AutoValue annotates constructor parameters that belong to `@Nullable` values with `@Nullable`.

So the generated autovalue class looks like this

``` java
$AutoValue_FoodDetailArgs(
      UUID productId,
      FoodTime foodTime,
      LocalDate date,
      @Nullable FoodDetailPresetServing foodDetailPresetServing) {
    if (productId == null) {
      throw new NullPointerException("Null productId");
    }
    this.productId = productId;
    if (foodTime == null) {
      throw new NullPointerException("Null foodTime");
    }
    this.foodTime = foodTime;
    if (date == null) {
      throw new NullPointerException("Null date");
    }
    this.date = date;
    this.foodDetailPresetServing = foodDetailPresetServing;
  }
```

When now autoparcel generates its implementation the annotations are not shown in its generated class:

``` java
  AutoValue_FoodDetailArgs(UUID productId, FoodTime foodTime, LocalDate date, FoodDetailPresetServing foodDetailPresetServing) {
    super(productId, foodTime, date, foodDetailPresetServing);
  }
```

**Comment by SUPERCILEX** [comment]
@rharter Pretty sure this was fixed at some point. The tests show constructors will nullability annotations: https://github.com/rharter/auto-value-parcel/blob/55e3b6405a9bb7b7fdb0bedb62a02fd4d2ad8435/auto-value-parcel/src/test/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtensionTest.java#L709

**Comment by rharter** [comment]
Agreed.  This was fixed in #124
